### PR TITLE
Allow var where color can be applied

### PR DESCRIPTION
**Pull request by rusty-key**
Noticed that using custom properties is not allowed for backgroundColor. This should cover the general needs for theming. 

But I think there should be a more generic solution, as `var` actually can be a part of any value:
https://developer.mozilla.org/en-US/docs/Web/CSS/var()

What do you think?

### CODE REVIEW

**Comment by giraud** [comment]
Same problem than with cascading.
I prefer some duplication in code instead of a complex type-based solution.

**Comment by rusty-key** [comment]
@giraud, by duplication you mean current solution?

File: bs-css/src/Css_Js_Core.rei [hunk 1]
```diff
@@ -1881,15 +1881,15 @@ let animationName: animationName => rule;
...
   let strokeLinejoin: [ | `miter | `round | `bevel] => rule;
   let strokeMiterlimit: float => rule;
   let strokeWidth: Types.Length.t => rule;
   let strokeOpacity: float => rule;
-  let stopColor: Types.Color.t => rule;
+  let stopColor: [ Types.Color.t | Types.Var.t ] => rule;
   let stopOpacity: float => rule;
-};
\ No newline at end of file
+};
```
Comment (giraud) [inline bs-css/src/Css_Js_Core.rei:1895]: missing `[<` here ?

**Comment by giraud** [comment]
@rusty-key yes

**Comment by rusty-key** [comment]
@giraud, fixed. Also added upper boundaries to gradients

**Comment by rusty-key** [comment]
Is there anything else I can do in this PR?

**Pull request merged**